apps/channels/datamodels.py

```diff
@@ -18,6 +18,30 @@ def content(self) -> str:
         return self.message_text
 
 
+class TelegramMessage(BaseModel):
+    chat_id: int = Field()
```

Comment:
Isn't having `Field()` the same as making it equal to `None`?

Reply:
No. It will still expet you to specify it, and will not allow the field to be `None`. Having `= Field()` is a bit redundant though

Reply:
What about the other instances of the same thing?

Reply:
:man_facepalming: https://github.com/dimagi/open-chat-studio/pull/261/commits/9a7c7e3456a98516d6e1d4d863b56440bb81135a